{"uuid": "7487293e-e8fa-4c4c-924a-ecc4a04dc5e5", "children": ["6e5d4c28-7ca8-41d1-9c69-21b596b6b33a"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\Desktop\\SW Testing\\Pytest framework\\CredKart_pytest_project 31-07-2023\\CredKart_Pytest_Project - Copy\\testCases\\conftest.py\", line 28, in setup\n    driver = webdriver.edge()\n             ^^^^^^^^^^^^^^^^\n"}, "start": 1691475798228, "stop": 1691475798228}], "start": 1691475798228, "stop": 1691475798234}